luigi_pipeline/lib/model/mito_mt_schema.py

```diff
@@ -0,0 +1,105 @@
+import hail as hl
+
+from lib.model.base_mt_schema import row_annotation
+from lib.model.seqr_mt_schema import BaseSeqrSchema, SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
+
+
+class SeqrMitoVariantSchema(BaseSeqrSchema):
+
+    def __init__(self, *args, high_constraint_region, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._high_constraint_region = high_constraint_region
+
+    # Mitochondrial only fields
+    @row_annotation()
+    def gnomad(self):
```

Comment:
depending on how Anne responds on the document, we may want to put this in a new `gnomad_mito` field, or we may want it to be the existing `gnomad_genomes` field, but until she responds we can't know which

Reply:
approval blocked on confirmation from Anne how we want to do this, will figure that out by tomorrow